semantic/src/site/globals/site.overrides

```diff
@@ -154,31 +154,42 @@ p strong {
   }
 }
 
-#application-area .ui.header .sub.header {
-  margin-top: 11px;
-  margin-bottom: 16px;
-  font-size: @fontLarge;
-  letter-spacing: @letterSpacingSmall;
+// Steps line (vertical box - layout)
+.line-vertical-box {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  min-height: 30px; // TODO: Need to be updated for mobile
+  align-items: center;
+  justify-content: space-between;
 }
 
-// Extra font and margin on Steps header on application home/create
-#application-area .steps-header {
-  font-size: @fontSmall;
-  font-weight: @regularWeight;
-  margin-top: 0;
-  margin-bottom: 5px;
+// Inner steps line (left side - icon & header)
+.line-vertical-box .centered-flex-box-row {
+  display: flex;
+  flex-grow: 1;
+  flex-direction: row;
+  align-items: center;
 }
 
-#application-area .ui.list > .item .header {
-  color: @darkGrey;
-  font-weight: @regularWeight;
-  font-size: @fontMedium;
+// Steps line - progress bar (box)
+.progress-box {
+  padding-right: 50px;
+  height: 3px;
 }
-#application-area .ui.list > .item > i.icon {
-  color: @darkGrey;
+
+// Steps line - progress bar
+.progress-box .progress {
+  width: 120px;
+}
+
+// Steps line - actions box
+.actions-box {
+  min-width: 100px;
 }
 
-#application-area .application-segment {
+// Message area segment - bottom of application home/create
+.application-segment {
   padding: 0;
 }
```

Comment:
This is _super_-generic: may as well call the class "padding-zero" and it can be used in lots of places

Reply:
Got it. : ) Done: 0ddcc68